WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -833,6 +839,20 @@ - (BlogDetailsRow *)peopleRow
     return row;
 }
 
+- (BlogDetailsRow *)usersRow
+{
+    __weak __typeof(self) weakSelf = self;
+    BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Users", @"Noun. Title. Links to the user management feature.")
+                                        accessibilityIdentifier:@"User Row"
+                                                          image:[UIImage imageNamed:@"site-menu-people"]
+                                                       callback:^{
+        if (@available(iOS 16.4, *)) {
```

Comment:
It doesn't seem there is anything preventing it from running on earlier versions.